docs/gh_pages/docs/retrieval_chain.md

```diff
@@ -203,3 +203,8 @@ Question: Please share the performance summary for John Smith?
 Answer: 
 I don't know, I'm sorry.
 ```
+
+## Prompt Governance
+When a user sends any prompt to the LLM using PebbloRetrieveQA, we capture the findings from that prompt. This behavior is enabled by default.
```

Comment:
As per Sridhar's guidelines, consider using the component name ("Pebblo" or "Pebblo Server") instead of "we", "you" etc